client/src/components/CardSpan.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { ICardSpanProps } from "../types";
+
+export default function Span({ title, value }: ICardSpanProps) {
```

Comment:
Why the name of component is different from the name of the file

Reply:
Mistake. Changed